src/Controller/UserController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class UserController
+{
+    /**
+     * @Route(path="/users")
+     */
+
+    public function getAllAction(Request $request): Response
+    {
+        return new Response("Hello Im Patricia", Response::HTTP_OK, []);
```

Comment:
Hello Patricia, I'd recommend using single quotes for simple strings like this. Using double quotes usually is a sign of a more complex usage. https://www.php.net/manual/en/language.types.string.php
For example:
$name = 'Patricia';
echo "Hello Im $name"; -> with double quotes will output: Hello Im Patricia
echo 'Hello Im $name'; -> with single quotes will output: Hello Im $name